contracts/src/v0.8/dev/keeper2_0/interfaces/OCR2Keeper.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+abstract contract OCR2Keeper {
```

Comment:
open Q. Do we really need this interface or can I combine this with KeeperRegistryInterface